Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/Ride/EndRide.hs

```diff
@@ -107,6 +113,17 @@ buildEndRideHandle merchantId = do
         getDistanceBetweenPoints = RideEndInt.getDistanceBetweenPoints merchantId
       }
 
+getFarePolicyByMerchantIdAndVariant :: Id DM.Merchant -> Variant -> Maybe Meters -> Flow (Maybe FarePolicy, Maybe SlabFarePolicy)
```

Comment:
Can we use here and in other places (like fare calculator) `Either FarePolicy SlabFarePolicy` instead of `(Maybe FarePolicy, Maybe SlabFarePolicy)`,  because only one item from this tuple required.

Or we can create some special data type like
`FarePolicy = NormalFP FarePolicy | SlabFP SlabFarePolicy`